### PR TITLE
Wayback Search component - add slot

### PR DESCRIPTION
**Pull request by iisa**
**Description**

adds optional slot to add optional branding next to wayback form

**Testing**

pull branch -> `npm install && npm start` -> demo starts w/ 2 search components, 1 with slot open

shrinking, floats under @ center.

### CODE REVIEW

File: packages/ia-wayback-search/index.html [hunk 1]
```diff
@@ -41,7 +41,21 @@
         waybackPagesArchived="32 trillion pages"
         .baseHost=${'https://archive.org'}
         .queryHandler=${queryHandler}
-      ></ia-wayback-search>`, document.getElementById('root'));
+        ></ia-wayback-search>
+        <ia-wayback-search
+          waybackPagesArchived="32 trillion pages"
```
Comment (bfalling) [inline packages/ia-wayback-search/index.html:46]: QUESTION: Is this the number Wayback Team instructed us to use? (q.v. @mc2 )

File: packages/ia-wayback-search/index.html [hunk 2]
```diff
@@ -41,7 +41,21 @@
...
+          waybackHost="https://archive.org"
+          .queryHandler=${queryHandler}
+          ?openSlot=${true}
+        >
+          <div slot="optional-branding">
+            <img
+              src="https://archive.org/download/ia-25-wf/wf-banner-button.png"
+              style="width: 100%; max-height: 75px;display: block;"
```
Comment (bfalling) [inline packages/ia-wayback-search/index.html:54]: SUGGESTION: Remove inline style?

File: packages/ia-wayback-search/stories/ia-wayback-search.stories.js [hunk 3]
```diff
@@ -38,5 +38,13 @@ storiesOf('<ia-wayback-search>', module)
         waybackPagesArchived="32 trillion pages"
         waybackHost="archive.org"
       ></ia-wayback-search>
+      <ia-wayback-search
+        waybackPagesArchived="32 trillion pages"
```
Comment (bfalling) [inline packages/ia-wayback-search/stories/ia-wayback-search.stories.js:42]: QUESTION: Verifying this number again. Wish there were a way to put this in a global constant that could be imported into all these components.

File: packages/ia-wayback-search/stories/ia-wayback-search.stories.js [hunk 4]
```diff
@@ -38,5 +38,13 @@ storiesOf('<ia-wayback-search>', module)
...
+        waybackHost="archive.org"
+      >
+        <div slot="optional-branding">
+          <img src="https://archive.org/download/ia-25-wf/wf-wb-search-square.png">
```
Comment (bfalling) [inline packages/ia-wayback-search/stories/ia-wayback-search.stories.js:46]: QUESTION: Same question as in other PR. Should we be hard-coding the production URL? Will that prevent easy testing on staging apps?

**Review by bfalling — COMMENTED** [review]
A few questions left. No blockers found.

**Comment by iisa** [comment]
closing as we are not using slot for prod. can reopen if needed again.

**Pull request closed without merging**